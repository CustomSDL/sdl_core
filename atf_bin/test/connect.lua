q = require("qttest")

obj1 = q.Object1()
dyn = qt.dynamic()
qt.connect(obj1, "Signal()", dyn, "TestSlot()")
qt.connect(obj1, "StringSignal(QString)", dyn, "TestStringSlot(QString)")
function dyn:TestSlot()
  print("TestSlot")
end
function dyn:TestStringSlot(str)
  print("TestStringSlot ", str)
  quit()
end
qt.connect(dyn, "Sig()", obj1, "Slot()")
qt.connect(dyn, "SigString(QString)", obj1, "StringSlot(QString)")
dyn:Sig()
dyn:SigString("Test string here")
obj1:raiseSignal()
obj1:raiseStringSignal("Test string here")
