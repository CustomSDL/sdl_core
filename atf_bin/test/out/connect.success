test.Object1 ctor
test.Object1 raiseSignal
test.Object1 raiseStringSignal
TestObject1::Slot()
TestObject1::StringSlot( "Test string here" )
TestSlot
TestStringSlot 	Test string here
test.Object1 __gc metamethod
TestObject::dtor
