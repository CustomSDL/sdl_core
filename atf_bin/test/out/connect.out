test/connect.lua:1: module 'qttest' not found:
	no field package.preload['qttest']
	no file './modules/qttest.lua'
	no file '/usr/local/share/lua/5.2/qttest.lua'
	no file '/usr/local/share/lua/5.2/qttest/init.lua'
	no file '/usr/local/lib/lua/5.2/qttest.lua'
	no file '/usr/local/lib/lua/5.2/qttest/init.lua'
	no file '/usr/share/lua/5.2/qttest.lua'
	no file '/usr/share/lua/5.2/qttest/init.lua'
	no file './qttest.lua'
	no file './modules/libqttest.so'
	no file './libqttest.so'
	no file '/usr/local/lib/lua/5.2/qttest.so'
	no file '/usr/lib/x86_64-linux-gnu/lua/5.2/qttest.so'
	no file '/usr/lib/lua/5.2/qttest.so'
	no file '/usr/local/lib/lua/5.2/loadall.so'
	no file './qttest.so'
